### PR TITLE
Feature navigation bar

### PR DESCRIPTION
**Pull request by smabhena**
Made use of a tab bar controller to set up the nav bar and navigation of the app to the main screens. Also fixed navigation between screens.

### CODE REVIEW

File: Mooov/Mooov/Controllers/LogInViewController.swift [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+//
+//  Created by Sinothando Mabhena on 2022/02/27.
+//
+
+import UIKit
+
+class LogInViewController: UIViewController {
+
+    @IBOutlet weak var usernameField: UITextField!
+
```
Comment (bgouws) [inline Mooov/Mooov/Controllers/LogInViewController.swift:13]: You can group these `@IBOutlets` together will make things a bit neater 😄 


```
@IBOutlet weak var usernameField: UITextField!
@IBOutlet weak var passwordField: UITextField!
```

File: Mooov/Mooov/Controllers/LogInViewController.swift [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            }
+            
+        } else {
+            self.displayAlert(title: "Log in failed",
+                              message: "Enter correct username or password",
+                              buttonTitle: "Ok")
+        }
+    }
+    
+    override func viewDidLoad() {
```
Comment (bgouws) [inline Mooov/Mooov/Controllers/LogInViewController.swift:32]: Just another code style comment, normally we like to keep our life cycle functions at the top of the class 😄

**Review by bgouws — CHANGES_REQUESTED** [review]
Just a little bit of clean up 😄

**Review by kirsty14 — APPROVED** [review]
Looking good! Keep up the good work

**Review by kirsty14 — APPROVED** [review]

**Review by bgouws — APPROVED** [review]
Happy from my side 😄

**Review by simekanidvt — APPROVED** [review]

**Pull request merged**